src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -16,29 +16,66 @@
 package net.sf.jabref.importer.fileformat;
 
 import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.Serializable;
 import java.util.ArrayList;
+
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Objects;
 
-import javax.xml.parsers.SAXParser;
-import javax.xml.parsers.SAXParserFactory;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
 
 import net.sf.jabref.importer.ParserResult;
 import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.IdGenerator;
 
+import com.google.common.base.Joiner;
+import generated.Abstract;
```

Comment:
I would use `net.sf.jabref.importer.fileformat.medline`. 
Reasons:
  -  Consistency with our ANTLR generation, which generates `src/main/gen/net/sf/jabref/bst/`. 
  - The common prefix `generated` does not indicate anyhting about medline. What if other XSDs will be generated?

Alternatively, use a combinded solution: `net.sf.jabref.importer.fileformat.medline.generated`.